includes/wikia/services/ApiService.class.php

```diff
@@ -112,6 +114,7 @@ private static function getHostByDbName( $dbName ) {
 	 * @return array $options
 	 */
 	public static function loginAsUser() {
```

Comment:
Please, add unit tests covering different `$options` returned by this method in different situations.